aerospike-mixin/dashboards/aerospike-overview.libsonnet

```diff
@@ -0,0 +1,1176 @@
+    {
+      id: 'sortBy',
+      options: {
+        fields: {},
+        sort: [
+          {
+            desc: true,
+            field: 'Time',
+          },
+        ],
+      },
+    },
+  ],
+};
+
+local clientReadsPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_read_success{aerospike_cluster=~"$aerospike_cluster", job=~"$job"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - success',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_read_error{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - error',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_read_filtered_out{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - filtered',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_read_timeout{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - timeout',
+    ),
+    prometheus.target(
+      'sum by(aerospike_cluster, job) (rate(aerospike_namespace_client_read_not_found{job=~"$job", aerospike_cluster=~"$aerospike_cluster"}[$__rate_interval]))',
+      datasource=promDatasource,
+      legendFormat='{{aerospike_cluster}} - not found',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Client reads',
+  description: 'Rate of client read transactions in an Aerospike cluster organized by result.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 20,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'linear',
+        lineWidth: 1,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'normal',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+          {
+            color: 'red',
+            value: 80,
+          },
+        ],
+      },
+      unit: 'ops',
```

Comment:
there are grafana units for "reads per second". not sure if you think that would be more appropriate